fire/cli/niv/_netoversigt.py

```diff
@@ -220,33 +183,80 @@ def path_to_origin(
     return None
 
 
-def analyser_subnet(net):
+def path_exists(net: dict[str, set[str]], start: str, origin: str) -> bool:
+    """
+    Bestem om net forbinder start til origin
+
+    Bruger breadth first search (BFS).
+    BFS Algoritme hugget fra: https://www.geeksforgeeks.org/breadth-first-search-or-bfs-for-a-graph/
```

Comment:
Det er uklart hvilke licensbetingelser der gælder for kode fra geeksforgeeks, så du må hellere omarbejde let og ændre kommentaren til "Baseret på materiale fra https://www...", så det ikke kommer i karambolage med at FIRE er MIT-licenseret.